internal/escape/escape.go

```diff
@@ -17,9 +17,24 @@ package escape
 
 import (
 	"fmt"
+	"net/url"
 	"strconv"
+	"strings"
 )
 
+// IsAlphanumeric returns true iff r is alphanumeric: a-z, A-Z, 0-9.
```

Comment:
Actually ASCII alphanumeric. Otherwise you should be using `unicode.IsLetter` and `unicode.IsNumber`.

Reply:
Done.